src/ucs/configure.m4

```diff
@@ -300,6 +300,23 @@ AS_IF([test "x$enable_builtin_memcpy" != xno],
 	  [AC_DEFINE([ENABLE_BUILTIN_MEMCPY], [0], [Enable builtin memcpy])]
   )
 
+#
+# Enable amd optimized buffer transfer
+#
+AC_ARG_ENABLE([amd-buffer-transfer],
+	AS_HELP_STRING([--enable-amd-buffer-transfer],
+	               [Enable amd optimized buffer transfer routine, default: NO]),
+	[],
+	[enable_amd_buffer_transfer=no])
+
+AS_IF([test "x$enable_amd_buffer_transfer" = xyes],
+	  [AS_MESSAGE([enabling amd optimized buffer transfer])
+	   AC_DEFINE([ENABLE_AMD_BUFFER_TRANSFER], [1], [Enable amd optimizedi buffer transfer])
+	   HAVE_AMD_BUFFER_TRANSFER=yes],
+	  [:]
+  )
```

Comment:
why not always enable it in compile-time?

Reply:
This method will not be suitable for transfers between cores that share a common L3cache, that is the reason I kept it as 'disabled' by default. 

Or can  I do like this?

a) Enable this feature always at compile time; Always disable at runtime
b) Use a ucx command-line variable (EG: -x UCX_ENABLE_DIRECTION_AWARE_COPY) to enable runtime.